C-Plus-Plus/cp/long_factorials.cpp

```diff
@@ -0,0 +1,39 @@
+/*Finding huge factorials such as !100, !1000 is sometimes required but in languages such as C++
+the datatypes are not even capable of storing such a big number. We could achieve calculation of
+such a number via going back to the basis of how we used to muiltiply two numbers back in school days. 
+We used to muiltiply numbers and would add the summation and get the final result. 
+This program here uses a vector to store each particular digit in a single index of the vector. 
+The carry is forwarded as basic maths and in the end the complete result gets stored in the vector*/
+
+#include <vector>
+#include <iostream>
+
+using namespace std;
+
+
+int main() {
```

Comment:
Use main() only for the driver code. Use some other function for the main code logic

Reply:
Done